docs-src/concepts/what-is-a-payload-codec.md

```diff
@@ -0,0 +1,43 @@
+---
+id: what-is-a-payload-codec
+title: What is a Payload Codec?
+sidebar_label: Payload Codec
+description: A Payload Codec transforms an array of Payloads (for example, a list of Workflow arguments) into another array of Payloads.
+tags:
+  - term
+  - explanation
+---
+
+A Payload Codec transforms an array of Payloads (for example, a list of Workflow arguments) into another array of Payloads.
+
+The Payload Codec is an optional step that happens between the wire and the Payload Converter:
+
+```bash
+Temporal Server <--> Wire <--> Payload Codec <--> Payload Converter <--> User code
+```
+
+When serializing to Payloads:
+
+- Data Converter is applied first, followed by the chain of codecs.
+- Codecs are applied last to first meaning the earlier encoders wrap the later ones.
```

Comment:
I would remove this. This is too deep and should be in javadocs. I would say that 

```
When serializing to Payloads:
- bla
- Chain of codec is applied 

When deserializing from Payloads:
- bla
- Chain of codecs is applied in a reversed order
```